packages/dd-trace/src/appsec/iast/analyzers/path-traversal-analyzer.js

```diff
@@ -29,7 +29,14 @@ class PathTraversalAnalyzer extends InjectionAnalyzer {
 
   onConfigure () {
     this.addSub('apm:fs:operation:start', (obj) => {
-      if (ignoredOperations.includes(obj.operation)) return
+      const store = storage.getStore()
+      const outOfReqOrChild = !store?.fs?.root
```

Comment:
if `store.fs` is undefined, we are not going to check the path traversal.
I think that it can be undefined if the subscription order is not the expected order.

Reply:
yes, `AppsecFsPlugin` must be subscribed before the analyzer.

we could remove this dependency if `AppsecFsPlugin` would publish another event...

Reply:
and could we add a test to ensure that we are calling to `AppsecFsPlugin.enable` before `enableAllAnalyzers`?